MAINTAINERS.yml

```diff
@@ -0,0 +1,1051 @@
+#         the given directory or directories. Otherwise, an exact match is
+#         required.
+#
+#         Paths to directories should always have a trailing '/'.
+#
+#     files-regex:
+#         List of regular expressions applied to paths to determine if they
+#         belong to the area. The regular expression may match anywhere within
+#         the path, but can be anchored with ^ and $ as usual.
+#
+#         Can be combined with a 'files' key.
+#
+#         Note: Prefer plain 'files' patterns where possible. get_maintainer.py
+#         will check that they match some file, but won't check regexes
+#         (because it might be slow).
+#
+#     files-exclude:
+#         Like 'files', but any matching files will be excluded from the area.
+#
+#     files-regex-exclude:
+#         Like 'files-regex', but any matching files will be excluded from the
+#         area.
+#
+#     description: >-
+#         Plain-English description. Describe what the system is about, from an
+#         outsider's perspective.
+#
+#
+# All areas must have a 'files' and/or 'files-regex' key. The other keys are
+# optional.
+#
+# It is very advisable to have a `status` key in all entries. Exceptions to
+# this would be sub-areas which add extra fields (for ex. more `collaborators`
+# who work only in that sub-area) to other areas.
+#
+#
+# Workflow
+# ########
+#
+# Ideally, any file in the tree will be covered by some area.
+#
+# When a GitHub pull request is sent, this happens:
+#
+#     * Users mentioned in 'maintainers' and 'collaborators' are added as
+#       reviewers to the pull request
+#
+#     * The labels listed in 'labels' are automatically added to the pull
+#       request
+#
+#     * The bot posts this comment:
+#
+#         This PR affects the following areas:
+#         <area name>:
+#           Status: ...
+#           Maintainers: <list of maintainers>
+#           Collaborators: <list of sub-maintainers>
+#           CC: <list of informed GitHub users, like @johndoe>
+#
+#         <area name>:
+#           ...
+#
+#     * An e-mail is sent to the addresses in 'inform'
+#
+# Changes to MAINTAINERS.yml need to be approved as follows:
+#
+#     * Changing the 'maintainers' for an area needs approval from the
+#       Technical Steering Committee
+#
+#     * Changing the 'collaborators' lines requires the maintainer and
+#       collaborators of that area to agree (or vote on it)
+
+# Areas are sorted by name
+
+"ARC arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/arc/
+        - include/arch/arc/
+    labels:
+        - "area: ARC"
+
+"ARM arch":
+    status: maintained
+    maintainers:
+        - ioannisg
+    collaborators:
+        - MaureenHelm
+        - galak
+    files:
+        - arch/arm/
+        - include/arch/arm/
+    labels:
+        - "area: ARM"
+
+Bluetooth:
```

Comment:
Doesn't this need " "?

Reply:
Nope, and not needed even if there's a space in it apparently. Pushed a clean-up commit that removes some more quotes.